### DISCUSSION TITLE
MC-Router API already in use; Errno(98): Address already in use

### DISCUSSION THREAD

**Discussion by rdebeuze** [fabianishere/udm-iptv#143] [General]
Hello,

I ran into an issue when reboot my UDM-P. I had an update from a old version that didn't work to i had to reinstall the IGMP proxy. Now i see this error MC-Router API already in use; Errno(98): Address already in use. So i think now there are 2 versions active the old one and the new version off the IGMP proxy. How can i delete the old one?

=== Service Logs ===
Jul 01 22:50:04 ubnt udm-iptv[20499]: Sending discover...
Jul 01 22:50:05 ubnt udm-iptv[20499]: Sending select for 10.134.162.159...
Jul 01 22:50:05 ubnt udm-iptv[20499]: Lease of 10.134.162.159 obtained, lease time 3600
Jul 01 22:50:05 ubnt udm-iptv[20499]: NATing IPTV network ranges (if necessary)
Jul 01 22:50:05 ubnt udm-iptv[20499]: Setting up igmpproxy
Jul 01 22:50:05 ubnt udm-iptv[20499]: Starting igmpproxy
Jul 01 22:50:05 ubnt udm-iptv[20499]: MC-Router API already in use; Errno(98): Address already in use
Jul 01 22:50:05 ubnt systemd[1]: udm-iptv.service: Main process exited, code=exited, status=255/EXCEPTION
Jul 01 22:50:05 ubnt systemd[1]: udm-iptv.service: Failed with result 'exit-code'.
Jul 01 22:50:05 ubnt systemd[1]: udm-iptv.service: Consumed 277ms CPU time.

Best Regards,
Ruben

**Comment by fabianishere** [comment]
Could you try:

```bash
systemctl disable --now igmpproxy
systemctl restart udm-iptv
```

**Reply by rdebeuze** [reply]
Thankyou for quick answer got this output:

root@ubnt:/# systemctl disable --now igmpproxy
igmpproxy.service is not a native service, redirecting to systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install disable igmpproxy

**Reply by fabianishere** [reply]
Does the service start correctly now?

**Comment by rdebeuze** [comment]
No same error:

=== Service Logs ===
Jul 01 23:01:19 ubnt udm-iptv[27188]: Sending discover...
Jul 01 23:01:19 ubnt udm-iptv[27188]: Sending select for 10.134.162.113...
Jul 01 23:01:19 ubnt udm-iptv[27188]: Lease of 10.134.162.113 obtained, lease time 3600
Jul 01 23:01:19 ubnt udm-iptv[27188]: NATing IPTV network ranges (if necessary)
Jul 01 23:01:19 ubnt udm-iptv[27188]: Setting up igmpproxy
Jul 01 23:01:19 ubnt udm-iptv[27188]: Starting igmpproxy
Jul 01 23:01:19 ubnt udm-iptv[27188]: MC-Router API already in use; Errno(98): Address already in use
Jul 01 23:01:19 ubnt systemd[1]: udm-iptv.service: Main process exited, code=exited, status=255/EXCEPTION
Jul 01 23:01:19 ubnt systemd[1]: udm-iptv.service: Failed with result 'exit-code'.
Jul 01 23:01:19 ubnt systemd[1]: udm-iptv.service: Consumed 239ms CPU time.

**Reply by fabianishere** [reply]
Did you use the old container-based setup before by any chance?

**Reply by fabianishere** [reply]
If so, remove the boot script and delete the container (in UbiOS):

```bash
podman stop iptv
podman rm iptv
```

**Reply by rdebeuze** [reply]
Yes did used that old container based setup. Thankyou very much the commands did work iptv works fine now!
